pom.xml

```diff
@@ -31,7 +31,7 @@
         <cucumber.version>7.2.3</cucumber.version>
         <jib-maven-plugin.version>3.1.1</jib-maven-plugin.version>
         <springfox-boot-starter.version>3.0.0</springfox-boot-starter.version>
-        <private-api-sdk-java.version>2.0.122</private-api-sdk-java.version>
+        <private-api-sdk-java.version>2.0.119</private-api-sdk-java.version>
```

Comment:
Can safely revert all the changes in `pom.xml` - I've added the fix to make the builds pass :)